--- libassuan-1.0.5/configure.ac.orig	2008-05-27 08:25:59.235870000 +0200
+++ libassuan-1.0.5/configure.ac	2008-05-27 08:36:37.868258393 +0200
@@ -44,7 +44,10 @@
 AB_INIT
 
 AC_GNU_SOURCE
-
+AH_TEMPLATE([_ALL_SOURCE], [All extensions])
+AH_TEMPLATE([_GNU_SOURCE], [GNU extensions])
+AH_TEMPLATE([_POSIX_PTHREAD_SEMANTICS], [POSIX pthread semantics])
+AH_TEMPLATE([_TANDEM_SOURCE], [Tandem extensions])
 
 AC_SUBST(PACKAGE)
 AC_SUBST(VERSION)
@@ -106,11 +109,6 @@
 have_dosish_system=no
 have_w32_system=no
 case "${host}" in
-    *-linux*)
-        if test "$GCC" = yes; then
-          CFLAGS="$CFLAGS -fPIC -DPIC"
-        fi
-        ;;
     *-mingw32*)
         have_dosish_system=yes
         have_w32_system=yes
