diff -Nru libassuan-1.0.5/configure.ac libassuan-1.0.5bs/configure.ac
--- libassuan-1.0.5/configure.ac	2010-03-30 15:41:46.678907652 +0200
+++ libassuan-1.0.5bs/configure.ac	2010-03-30 15:41:12.185577693 +0200
@@ -29,7 +29,7 @@
 
 m4_define([svn_revision], m4_esyscmd([echo -n $( (svn info 2>/dev/null \
             || echo 'Revision: 0')|sed -n '/^Revision:/ {s/[^0-9]//gp;q;}')]))
-AC_INIT([libassuan], my_version[]m4_if(my_issvn,[yes],[-svn[]svn_revision]),
+AC_INIT([libassuan1], my_version[]m4_if(my_issvn,[yes],[-svn[]svn_revision]),
         [bug-libassuan@gnupg.org])
 # Note, that this is not yet available as a shared library.

@@ -148,7 +146,7 @@
 AC_SUBST(NETLIBS)

 # For src/libassuan-config.in
-LIBASSUAN_CONFIG_LIB="-lassuan"
+LIBASSUAN_CONFIG_LIB="-lassuan1"
 LIBASSUAN_CONFIG_CFLAGS=""
 LIBASSUAN_CONFIG_API_VERSION="1"
 LIBASSUAN_CONFIG_THREAD_MODULES=
@@ -263,6 +263,6 @@
 AC_CONFIG_FILES([src/Makefile])
 AC_CONFIG_FILES([doc/Makefile])
 AC_CONFIG_FILES([tests/Makefile])
-AC_CONFIG_FILES([src/libassuan-config], [chmod +x src/libassuan-config])
+AC_CONFIG_FILES([src/libassuan1-config], [chmod +x src/libassuan1-config])
 
 AC_OUTPUT
diff -Nru libassuan-1.0.5/src/libassuan1-config.in libassuan-1.0.5bs/src/libassuan1-config.in
--- libassuan-1.0.5/src/libassuan1-config.in	1970-01-01 01:00:00.000000000 +0100
+++ libassuan-1.0.5bs/src/libassuan1-config.in	2010-03-30 15:40:42.385580650 +0200
@@ -0,0 +1,144 @@
+#!/bin/sh
+#    Copyright (C) 1999, 2002, 2003 Free Software Foundation, Inc.
+#
+# This file is free software; as a special exception the author gives
+# unlimited permission to copy and/or distribute it, with or without
+# modifications, as long as this notice is preserved.
+#
+# This file is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY, to the extent permitted by law; without even the
+# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+PGM=libassuan1-config
+lib="@LIBASSUAN_CONFIG_LIB@"
+extralibs="@LIBASSUAN_CONFIG_EXTRA_LIBS@"
+cflags="@LIBASSUAN_CONFIG_CFLAGS@"
+api_version="@LIBASSUAN_CONFIG_API_VERSION@"
+all_thread_modules="@LIBASSUAN_CONFIG_THREAD_MODULES@"
+thread_module=
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+includes=""
+libdirs=""
+exec_prefix_set=no
+echo_libs=no
+echo_cflags=no
+echo_prefix=no
+echo_exec_prefix=no
+
+if test x"$all_thread_modules" = x; then
+  all_thread_modules="none pthread"
+else
+  all_thread_modules="none pthread $all_thread_modules"
+fi
+
+usage()
+{
+	cat <<EOF
+Usage: $PGM [OPTIONS]
+Options:
+	[--thread={`echo "${all_thread_modules}" | sed 's/ /|/g'`}]
+	[--prefix[=DIR]]
+	[--exec-prefix[=DIR]]
+	[--version]
+	[--libs]
+	[--cflags]
+EOF
+	exit $1
+}
+
+if test $# -eq 0; then
+  usage 1 1>&2
+fi
+
+while test $# -gt 0; do
+  case "$1" in
+  -*=*) optarg=`echo "$1" | sed 's/[-_a-zA-Z0-9]*=//'` ;;
+  *) optarg= ;;
+  esac
+
+  case $1 in
+    --prefix=*)
+      # Dummy
+      ;;
+    --prefix)
+      echo_prefix=yes
+      ;;
+    --exec-prefix=*)
+      # Dummy
+      ;;
+    --exec-prefix)
+      echo_exec_prefix=yes
+      ;;
+    --version)
+      echo "@VERSION@"
+      exit 0
+      ;;
+    --api-version)
+      echo_api_version=yes
+      ;;
+    --thread=*)
+      for mod in $all_thread_modules; do
+	if test "$mod" = "$optarg"; then
+           thread_module="-$mod"
+	fi
+      done
+      if test "x$thread_module" = "x"; then
+	usage 1 1>&2
+      fi
+      if test "$thread_module" = "-none"; then
+        thread_module=""
+      fi
+      if test "$thread_module" = "-pthread"; then
+        thread_module=""
+      fi
+      ;;
+    --cflags)
+      echo_cflags=yes
+      ;;
+    --libs)
+      echo_libs=yes
+      ;;
+    *)
+      usage 1 1>&2
+      ;;
+  esac
+  shift
+done
+
+if test "$echo_prefix" = "yes"; then
+    echo $prefix
+fi
+
+if test "$echo_exec_prefix" = "yes"; then
+    echo $exec_prefix
+fi
+
+if test "$echo_api_version" = "yes"; then
+    echo $api_version
+fi
+
+
+if test "$echo_cflags" = "yes"; then
+    if test "@includedir@" != "/usr/include" ; then
+      includes="-I@includedir@"
+      for i in $cflags ; do
+	if test "$i" = "-I@includedir@" ; then
+	  includes=""
+	fi
+      done
+    fi
+    echo $includes $cflags
+fi
+
+if test "$echo_libs" = "yes"; then
+    if test "@libdir@" != "/usr/lib" ; then
+      libdirs="-L@libdir@"
+      for i in $lib $extralibs ; do
+	if test "$i" = "-L@libdir@" ; then
+	  libdirs=""
+	fi
+      done
+    fi
+    echo $libdirs $lib${thread_module} $extralibs
+fi
diff -Nru libassuan-1.0.5/src/libassuan1.m4 libassuan-1.0.5bs/src/libassuan1.m4
--- libassuan-1.0.5/src/libassuan1.m4	1970-01-01 01:00:00.000000000 +0100
+++ libassuan-1.0.5bs/src/libassuan1.m4	2007-05-09 11:29:14.000000000 +0200
@@ -0,0 +1,175 @@
+dnl Autoconf macros for libassuan
+dnl       Copyright (C) 2002, 2003 Free Software Foundation, Inc.
+dnl
+dnl This file is free software; as a special exception the author gives
+dnl unlimited permission to copy and/or distribute it, with or without
+dnl modifications, as long as this notice is preserved.
+dnl
+dnl This file is distributed in the hope that it will be useful, but
+dnl WITHOUT ANY WARRANTY, to the extent permitted by law; without even the
+dnl implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+dnl
+dnl Common code used for libassuan detection [internal]
+dnl Returns ok set to yes or no.
+dnl
+AC_DEFUN([_AM_PATH_LIBASSUAN_COMMON],
+[ AC_ARG_WITH(libassuan-prefix,
+              AC_HELP_STRING([--with-libassuan-prefix=PFX],
+                             [prefix where LIBASSUAN is installed (optional)]),
+     libassuan_config_prefix="$withval", libassuan_config_prefix="")
+  if test x$libassuan_config_prefix != x ; then
+    libassuan_config_args="$libassuan_config_args --prefix=$libassuan_config_prefix"
+    if test x${LIBASSUAN_CONFIG+set} != xset ; then
+      LIBASSUAN_CONFIG=$libassuan_config_prefix/bin/libassuan-config
+    fi
+  fi
+  AC_PATH_PROG(LIBASSUAN_CONFIG, libassuan-config, no)
+
+  tmp=ifelse([$1], ,1:0.9.2,$1)
+  if echo "$tmp" | grep ':' >/dev/null 2>/dev/null ; then
+    req_libassuan_api=`echo "$tmp"     | sed 's/\(.*\):\(.*\)/\1/'`
+    min_libassuan_version=`echo "$tmp" | sed 's/\(.*\):\(.*\)/\2/'`
+  else
+    req_libassuan_api=0
+    min_libassuan_version="$tmp"
+  fi
+
+  if test "$LIBASSUAN_CONFIG" != "no" ; then
+    libassuan_version=`$LIBASSUAN_CONFIG --version`
+  fi
+  libassuan_version_major=`echo $libassuan_version | \
+               sed 's/\([[0-9]]*\)\.\([[0-9]]*\)\.\([[0-9]]*\).*/\1/'`
+  libassuan_version_minor=`echo $libassuan_version | \
+               sed 's/\([[0-9]]*\)\.\([[0-9]]*\)\.\([[0-9]]*\).*/\2/'`
+  libassuan_version_micro=`echo $libassuan_version | \
+               sed 's/\([[0-9]]*\)\.\([[0-9]]*\)\.\([[0-9]]*\).*/\3/'`
+
+  AC_MSG_CHECKING(for LIBASSUAN ifelse([$2], ,,[$2 ])- version >= $min_libassuan_version)
+  ok=no
+  if test "$LIBASSUAN_CONFIG" != "no" ; then
+    ifelse([$2], ,,[if `$LIBASSUAN_CONFIG --thread=$2 2> /dev/null` ; then])
+    req_major=`echo $min_libassuan_version | \
+               sed 's/\([[0-9]]*\)\.\([[0-9]]*\)\.\([[0-9]]*\)/\1/'`
+    req_minor=`echo $min_libassuan_version | \
+               sed 's/\([[0-9]]*\)\.\([[0-9]]*\)\.\([[0-9]]*\)/\2/'`
+    req_micro=`echo $min_libassuan_version | \
+               sed 's/\([[0-9]]*\)\.\([[0-9]]*\)\.\([[0-9]]*\)/\3/'`
+    if test "$libassuan_version_major" -gt "$req_major"; then
+        ok=yes
+    else 
+        if test "$libassuan_version_major" -eq "$req_major"; then
+            if test "$libassuan_version_minor" -gt "$req_minor"; then
+               ok=yes
+            else
+               if test "$libassuan_version_minor" -eq "$req_minor"; then
+                   if test "$libassuan_version_micro" -ge "$req_micro"; then
+                     ok=yes
+                   fi
+               fi
+            fi
+        fi
+    fi
+    ifelse([$2], ,,[fi])
+  fi
+
+  if test $ok = yes; then
+    AC_MSG_RESULT([yes ($libassuan_version)])
+  else
+    AC_MSG_RESULT(no)
+  fi
+
+  if test $ok = yes; then
+    if test "$req_libassuan_api" -gt 0 ; then
+      tmp=`$LIBASSUAN_CONFIG --api-version 2>/dev/null || echo 0`
+      if test "$tmp" -gt 0 ; then
+        AC_MSG_CHECKING([LIBASSUAN ifelse([$2], ,,[$2 ])API version])
+        if test "$req_libassuan_api" -eq "$tmp" ; then
+          AC_MSG_RESULT(okay)
+        else
+          ok=no
+          AC_MSG_RESULT([does not match.  want=$req_libassuan_api got=$tmp.])
+        fi
+      fi
+    fi
+  fi
+
+])
+
+dnl AM_CHECK_LIBASSUAN([MINIMUM-VERSION,
+dnl                    [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND ]]])
+dnl Test whether libassuan has at least MINIMUM-VERSION. This is
+dnl used to test for features only available in newer versions.
+dnl
+AC_DEFUN([AM_CHECK_LIBASSUAN],
+[ _AM_PATH_LIBASSUAN_COMMON($1)
+  if test $ok = yes; then
+    ifelse([$2], , :, [$2])
+  else
+    ifelse([$3], , :, [$3])
+  fi
+])
+
+
+
+
+dnl AM_PATH_LIBASSUAN([MINIMUM-VERSION,
+dnl                   [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND ]]])
+dnl Test for libassuan and define LIBASSUAN_CFLAGS and LIBASSUAN_LIBS
+dnl
+AC_DEFUN([AM_PATH_LIBASSUAN],
+[ _AM_PATH_LIBASSUAN_COMMON($1)
+  if test $ok = yes; then
+    LIBASSUAN_CFLAGS=`$LIBASSUAN_CONFIG $libassuan_config_args --cflags`
+    LIBASSUAN_LIBS=`$LIBASSUAN_CONFIG $libassuan_config_args --libs`
+    ifelse([$2], , :, [$2])
+  else
+    LIBASSUAN_CFLAGS=""
+    LIBASSUAN_LIBS=""
+    ifelse([$3], , :, [$3])
+  fi
+  AC_SUBST(LIBASSUAN_CFLAGS)
+  AC_SUBST(LIBASSUAN_LIBS)
+])
+
+
+dnl AM_PATH_LIBASSUAN_PTH([MINIMUM-VERSION,
+dnl                      [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND ]]])
+dnl Test for libassuan and define LIBASSUAN_PTH_CFLAGS and LIBASSUAN_PTH_LIBS
+dnl
+AC_DEFUN([AM_PATH_LIBASSUAN_PTH],
+[ _AM_PATH_LIBASSUAN_COMMON($1,pth)
+  if test $ok = yes; then
+    LIBASSUAN_PTH_CFLAGS=`$LIBASSUAN_CONFIG $libassuan_config_args --thread=pth --cflags`
+    LIBASSUAN_PTH_LIBS=`$LIBASSUAN_CONFIG $libassuan_config_args --thread=pth --libs`
+    ifelse([$2], , :, [$2])
+  else
+    LIBASSUAN_PTH_CFLAGS=""
+    LIBASSUAN_PTH_LIBS=""
+    ifelse([$3], , :, [$3])
+  fi
+  AC_SUBST(LIBASSUAN_PTH_CFLAGS)
+  AC_SUBST(LIBASSUAN_PTH_LIBS)
+])
+
+
+dnl AM_PATH_LIBASSUAN_PTHREAD([MINIMUM-VERSION,
+dnl                           [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND ]]])
+dnl Test for libassuan and define LIBASSUAN_PTHREAD_CFLAGS 
+dnl                           and LIBASSUAN_PTHREAD_LIBS
+dnl
+AC_DEFUN([AM_PATH_LIBASSUAN_PTHREAD],
+[ _AM_PATH_LIBASSUAN_COMMON($1,pthread)
+  if test $ok = yes; then
+    LIBASSUAN_PTHREAD_CFLAGS=`$LIBASSUAN_CONFIG $libassuan_config_args --thread=pthread --cflags`
+    LIBASSUAN_PTHREAD_LIBS=`$LIBASSUAN_CONFIG $libassuan_config_args --thread=pthread --libs`
+    ifelse([$2], , :, [$2])
+  else
+    LIBASSUAN_PTHREAD_CFLAGS=""
+    LIBASSUAN_PTHREAD_LIBS=""
+    ifelse([$3], , :, [$3])
+  fi
+  AC_SUBST(LIBASSUAN_PTHREAD_CFLAGS)
+  AC_SUBST(LIBASSUAN_PTHREAD_LIBS)
+])
+
diff -Nru libassuan-1.0.5/src/libassuan-config.in libassuan-1.0.5bs/src/libassuan-config.in
--- libassuan-1.0.5/src/libassuan-config.in	2006-10-09 12:41:47.000000000 +0200
+++ libassuan-1.0.5bs/src/libassuan-config.in	1970-01-01 01:00:00.000000000 +0100
@@ -1,144 +0,0 @@
-#!/bin/sh
-#    Copyright (C) 1999, 2002, 2003 Free Software Foundation, Inc.
-#
-# This file is free software; as a special exception the author gives
-# unlimited permission to copy and/or distribute it, with or without
-# modifications, as long as this notice is preserved.
-#
-# This file is distributed in the hope that it will be useful, but
-# WITHOUT ANY WARRANTY, to the extent permitted by law; without even the
-# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-
-PGM=libassuan-config
-lib="@LIBASSUAN_CONFIG_LIB@"
-extralibs="@LIBASSUAN_CONFIG_EXTRA_LIBS@"
-cflags="@LIBASSUAN_CONFIG_CFLAGS@"
-api_version="@LIBASSUAN_CONFIG_API_VERSION@"
-all_thread_modules="@LIBASSUAN_CONFIG_THREAD_MODULES@"
-thread_module=
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-includes=""
-libdirs=""
-exec_prefix_set=no
-echo_libs=no
-echo_cflags=no
-echo_prefix=no
-echo_exec_prefix=no
-
-if test x"$all_thread_modules" = x; then
-  all_thread_modules="none pthread"
-else
-  all_thread_modules="none pthread $all_thread_modules"
-fi
-
-usage()
-{
-	cat <<EOF
-Usage: $PGM [OPTIONS]
-Options:
-	[--thread={`echo "${all_thread_modules}" | sed 's/ /|/g'`}]
-	[--prefix[=DIR]]
-	[--exec-prefix[=DIR]]
-	[--version]
-	[--libs]
-	[--cflags]
-EOF
-	exit $1
-}
-
-if test $# -eq 0; then
-  usage 1 1>&2
-fi
-
-while test $# -gt 0; do
-  case "$1" in
-  -*=*) optarg=`echo "$1" | sed 's/[-_a-zA-Z0-9]*=//'` ;;
-  *) optarg= ;;
-  esac
-
-  case $1 in
-    --prefix=*)
-      # Dummy
-      ;;
-    --prefix)
-      echo_prefix=yes
-      ;;
-    --exec-prefix=*)
-      # Dummy
-      ;;
-    --exec-prefix)
-      echo_exec_prefix=yes
-      ;;
-    --version)
-      echo "@VERSION@"
-      exit 0
-      ;;
-    --api-version)
-      echo_api_version=yes
-      ;;
-    --thread=*)
-      for mod in $all_thread_modules; do
-	if test "$mod" = "$optarg"; then
-           thread_module="-$mod"
-	fi
-      done
-      if test "x$thread_module" = "x"; then
-	usage 1 1>&2
-      fi
-      if test "$thread_module" = "-none"; then
-        thread_module=""
-      fi
-      if test "$thread_module" = "-pthread"; then
-        thread_module=""
-      fi
-      ;;
-    --cflags)
-      echo_cflags=yes
-      ;;
-    --libs)
-      echo_libs=yes
-      ;;
-    *)
-      usage 1 1>&2
-      ;;
-  esac
-  shift
-done
-
-if test "$echo_prefix" = "yes"; then
-    echo $prefix
-fi
-
-if test "$echo_exec_prefix" = "yes"; then
-    echo $exec_prefix
-fi
-
-if test "$echo_api_version" = "yes"; then
-    echo $api_version
-fi
-
-
-if test "$echo_cflags" = "yes"; then
-    if test "@includedir@" != "/usr/include" ; then
-      includes="-I@includedir@"
-      for i in $cflags ; do
-	if test "$i" = "-I@includedir@" ; then
-	  includes=""
-	fi
-      done
-    fi
-    echo $includes $cflags
-fi
-
-if test "$echo_libs" = "yes"; then
-    if test "@libdir@" != "/usr/lib" ; then
-      libdirs="-L@libdir@"
-      for i in $lib $extralibs ; do
-	if test "$i" = "-L@libdir@" ; then
-	  libdirs=""
-	fi
-      done
-    fi
-    echo $libdirs $lib${thread_module} $extralibs
-fi
diff -Nru libassuan-1.0.5/src/libassuan.m4 libassuan-1.0.5bs/src/libassuan.m4
--- libassuan-1.0.5/src/libassuan.m4	2007-05-09 11:29:14.000000000 +0200
+++ libassuan-1.0.5bs/src/libassuan.m4	1970-01-01 01:00:00.000000000 +0100
@@ -1,175 +0,0 @@
-dnl Autoconf macros for libassuan
-dnl       Copyright (C) 2002, 2003 Free Software Foundation, Inc.
-dnl
-dnl This file is free software; as a special exception the author gives
-dnl unlimited permission to copy and/or distribute it, with or without
-dnl modifications, as long as this notice is preserved.
-dnl
-dnl This file is distributed in the hope that it will be useful, but
-dnl WITHOUT ANY WARRANTY, to the extent permitted by law; without even the
-dnl implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-
-dnl
-dnl Common code used for libassuan detection [internal]
-dnl Returns ok set to yes or no.
-dnl
-AC_DEFUN([_AM_PATH_LIBASSUAN_COMMON],
-[ AC_ARG_WITH(libassuan-prefix,
-              AC_HELP_STRING([--with-libassuan-prefix=PFX],
-                             [prefix where LIBASSUAN is installed (optional)]),
-     libassuan_config_prefix="$withval", libassuan_config_prefix="")
-  if test x$libassuan_config_prefix != x ; then
-    libassuan_config_args="$libassuan_config_args --prefix=$libassuan_config_prefix"
-    if test x${LIBASSUAN_CONFIG+set} != xset ; then
-      LIBASSUAN_CONFIG=$libassuan_config_prefix/bin/libassuan-config
-    fi
-  fi
-  AC_PATH_PROG(LIBASSUAN_CONFIG, libassuan-config, no)
-
-  tmp=ifelse([$1], ,1:0.9.2,$1)
-  if echo "$tmp" | grep ':' >/dev/null 2>/dev/null ; then
-    req_libassuan_api=`echo "$tmp"     | sed 's/\(.*\):\(.*\)/\1/'`
-    min_libassuan_version=`echo "$tmp" | sed 's/\(.*\):\(.*\)/\2/'`
-  else
-    req_libassuan_api=0
-    min_libassuan_version="$tmp"
-  fi
-
-  if test "$LIBASSUAN_CONFIG" != "no" ; then
-    libassuan_version=`$LIBASSUAN_CONFIG --version`
-  fi
-  libassuan_version_major=`echo $libassuan_version | \
-               sed 's/\([[0-9]]*\)\.\([[0-9]]*\)\.\([[0-9]]*\).*/\1/'`
-  libassuan_version_minor=`echo $libassuan_version | \
-               sed 's/\([[0-9]]*\)\.\([[0-9]]*\)\.\([[0-9]]*\).*/\2/'`
-  libassuan_version_micro=`echo $libassuan_version | \
-               sed 's/\([[0-9]]*\)\.\([[0-9]]*\)\.\([[0-9]]*\).*/\3/'`
-
-  AC_MSG_CHECKING(for LIBASSUAN ifelse([$2], ,,[$2 ])- version >= $min_libassuan_version)
-  ok=no
-  if test "$LIBASSUAN_CONFIG" != "no" ; then
-    ifelse([$2], ,,[if `$LIBASSUAN_CONFIG --thread=$2 2> /dev/null` ; then])
-    req_major=`echo $min_libassuan_version | \
-               sed 's/\([[0-9]]*\)\.\([[0-9]]*\)\.\([[0-9]]*\)/\1/'`
-    req_minor=`echo $min_libassuan_version | \
-               sed 's/\([[0-9]]*\)\.\([[0-9]]*\)\.\([[0-9]]*\)/\2/'`
-    req_micro=`echo $min_libassuan_version | \
-               sed 's/\([[0-9]]*\)\.\([[0-9]]*\)\.\([[0-9]]*\)/\3/'`
-    if test "$libassuan_version_major" -gt "$req_major"; then
-        ok=yes
-    else 
-        if test "$libassuan_version_major" -eq "$req_major"; then
-            if test "$libassuan_version_minor" -gt "$req_minor"; then
-               ok=yes
-            else
-               if test "$libassuan_version_minor" -eq "$req_minor"; then
-                   if test "$libassuan_version_micro" -ge "$req_micro"; then
-                     ok=yes
-                   fi
-               fi
-            fi
-        fi
-    fi
-    ifelse([$2], ,,[fi])
-  fi
-
-  if test $ok = yes; then
-    AC_MSG_RESULT([yes ($libassuan_version)])
-  else
-    AC_MSG_RESULT(no)
-  fi
-
-  if test $ok = yes; then
-    if test "$req_libassuan_api" -gt 0 ; then
-      tmp=`$LIBASSUAN_CONFIG --api-version 2>/dev/null || echo 0`
-      if test "$tmp" -gt 0 ; then
-        AC_MSG_CHECKING([LIBASSUAN ifelse([$2], ,,[$2 ])API version])
-        if test "$req_libassuan_api" -eq "$tmp" ; then
-          AC_MSG_RESULT(okay)
-        else
-          ok=no
-          AC_MSG_RESULT([does not match.  want=$req_libassuan_api got=$tmp.])
-        fi
-      fi
-    fi
-  fi
-
-])
-
-dnl AM_CHECK_LIBASSUAN([MINIMUM-VERSION,
-dnl                    [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND ]]])
-dnl Test whether libassuan has at least MINIMUM-VERSION. This is
-dnl used to test for features only available in newer versions.
-dnl
-AC_DEFUN([AM_CHECK_LIBASSUAN],
-[ _AM_PATH_LIBASSUAN_COMMON($1)
-  if test $ok = yes; then
-    ifelse([$2], , :, [$2])
-  else
-    ifelse([$3], , :, [$3])
-  fi
-])
-
-
-
-
-dnl AM_PATH_LIBASSUAN([MINIMUM-VERSION,
-dnl                   [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND ]]])
-dnl Test for libassuan and define LIBASSUAN_CFLAGS and LIBASSUAN_LIBS
-dnl
-AC_DEFUN([AM_PATH_LIBASSUAN],
-[ _AM_PATH_LIBASSUAN_COMMON($1)
-  if test $ok = yes; then
-    LIBASSUAN_CFLAGS=`$LIBASSUAN_CONFIG $libassuan_config_args --cflags`
-    LIBASSUAN_LIBS=`$LIBASSUAN_CONFIG $libassuan_config_args --libs`
-    ifelse([$2], , :, [$2])
-  else
-    LIBASSUAN_CFLAGS=""
-    LIBASSUAN_LIBS=""
-    ifelse([$3], , :, [$3])
-  fi
-  AC_SUBST(LIBASSUAN_CFLAGS)
-  AC_SUBST(LIBASSUAN_LIBS)
-])
-
-
-dnl AM_PATH_LIBASSUAN_PTH([MINIMUM-VERSION,
-dnl                      [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND ]]])
-dnl Test for libassuan and define LIBASSUAN_PTH_CFLAGS and LIBASSUAN_PTH_LIBS
-dnl
-AC_DEFUN([AM_PATH_LIBASSUAN_PTH],
-[ _AM_PATH_LIBASSUAN_COMMON($1,pth)
-  if test $ok = yes; then
-    LIBASSUAN_PTH_CFLAGS=`$LIBASSUAN_CONFIG $libassuan_config_args --thread=pth --cflags`
-    LIBASSUAN_PTH_LIBS=`$LIBASSUAN_CONFIG $libassuan_config_args --thread=pth --libs`
-    ifelse([$2], , :, [$2])
-  else
-    LIBASSUAN_PTH_CFLAGS=""
-    LIBASSUAN_PTH_LIBS=""
-    ifelse([$3], , :, [$3])
-  fi
-  AC_SUBST(LIBASSUAN_PTH_CFLAGS)
-  AC_SUBST(LIBASSUAN_PTH_LIBS)
-])
-
-
-dnl AM_PATH_LIBASSUAN_PTHREAD([MINIMUM-VERSION,
-dnl                           [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND ]]])
-dnl Test for libassuan and define LIBASSUAN_PTHREAD_CFLAGS 
-dnl                           and LIBASSUAN_PTHREAD_LIBS
-dnl
-AC_DEFUN([AM_PATH_LIBASSUAN_PTHREAD],
-[ _AM_PATH_LIBASSUAN_COMMON($1,pthread)
-  if test $ok = yes; then
-    LIBASSUAN_PTHREAD_CFLAGS=`$LIBASSUAN_CONFIG $libassuan_config_args --thread=pthread --cflags`
-    LIBASSUAN_PTHREAD_LIBS=`$LIBASSUAN_CONFIG $libassuan_config_args --thread=pthread --libs`
-    ifelse([$2], , :, [$2])
-  else
-    LIBASSUAN_PTHREAD_CFLAGS=""
-    LIBASSUAN_PTHREAD_LIBS=""
-    ifelse([$3], , :, [$3])
-  fi
-  AC_SUBST(LIBASSUAN_PTHREAD_CFLAGS)
-  AC_SUBST(LIBASSUAN_PTHREAD_LIBS)
-])
-
diff -Nru libassuan-1.0.5/src/Makefile.am libassuan-1.0.5bs/src/Makefile.am
--- libassuan-1.0.5/src/Makefile.am	2010-03-30 15:41:46.675574522 +0200
+++ libassuan-1.0.5bs/src/Makefile.am	2010-03-30 15:39:55.805585239 +0200
@@ -17,21 +17,21 @@
 # License along with this program; if not, see <http://www.gnu.org/licenses/>.
 ## Process this file with automake to produce Makefile.in
 
-EXTRA_DIST = libassuan-config.in mkerrors libassuan.m4
+EXTRA_DIST = libassuan1-config.in mkerrors libassuan1.m4
 INCLUDES = -I.. -I$(top_srcdir)/include
 BUILT_SOURCES = assuan-errors.c
 MOSTLYCLEANFILES = assuan-errors.c
 
 if HAVE_PTH
-libassuan_pth = libassuan-pth.la
+libassuan1_pth = libassuan1-pth.la
 else
-libassuan_pth = 
+libassuan1_pth = 
 endif
 
-bin_SCRIPTS = libassuan-config
+bin_SCRIPTS = libassuan1-config
 m4datadir = $(datadir)/aclocal
-m4data_DATA = libassuan.m4
-lib_LTLIBRARIES = libassuan.la $(libassuan_pth)
+m4data_DATA = libassuan1.m4
+lib_LTLIBRARIES = libassuan1.la $(libassuan1_pth)
 include_HEADERS = assuan.h
 
 
@@ -53,13 +53,13 @@
 	assuan-logging.c \
 	assuan-socket.c
 
-libassuan_la_SOURCES = $(common_sources) assuan-io.c
-libassuan_la_LIBADD = @LTLIBOBJS@
+libassuan1_la_SOURCES = $(common_sources) assuan-io.c
+libassuan1_la_LIBADD = @LTLIBOBJS@
 
 if HAVE_PTH
-libassuan_pth_la_SOURCES = $(common_sources) assuan-io-pth.c
-libassuan_pth_la_CFLAGS = $(AM_CFLAGS) $(PTH_CFLAGS)
-libassuan_pth_la_LIBADD = @LTLIBOBJS@ $(PTH_LIBS)
+libassuan1_pth_la_SOURCES = $(common_sources) assuan-io-pth.c
+libassuan1_pth_la_CFLAGS = $(AM_CFLAGS) $(PTH_CFLAGS)
+libassuan1_pth_la_LIBADD = @LTLIBOBJS@ $(PTH_LIBS)
 endif
 
 assuan-errors.c : assuan.h mkerrors
--- libassuan-1.0.5/tests/Makefile.am.orig	2010-03-30 15:42:49.061919784 +0200
+++ libassuan-1.0.5/tests/Makefile.am	2010-03-30 15:44:46.594515962 +0200
@@ -35,5 +35,5 @@
 
 noinst_HEADERS = common.h
 noinst_PROGRAMS = $(TESTS)
-LDADD =  ../src/libassuan.la  $(NETLIBS) # $(GPG_ERROR_LIBS)
+LDADD =  ../src/libassuan1.la  $(NETLIBS) # $(GPG_ERROR_LIBS)
 
