diff -Nru libassuan-1.0.5/configure.ac libassuan-1.0.5bs/configure.ac
--- libassuan-1.0.5/configure.ac	2010-03-30 15:41:46.678907652 +0200
+++ libassuan-1.0.5bs/configure.ac	2010-03-30 15:41:12.185577693 +0200
@@ -29,7 +29,7 @@
 
 m4_define([svn_revision], m4_esyscmd([echo -n $( (svn info 2>/dev/null \
             || echo 'Revision: 0')|sed -n '/^Revision:/ {s/[^0-9]//gp;q;}')]))
-AC_INIT([libassuan], my_version[]m4_if(my_issvn,[yes],[-svn[]svn_revision]),
+AC_INIT([libassuan1], my_version[]m4_if(my_issvn,[yes],[-svn[]svn_revision]),
         [bug-libassuan@gnupg.org])
 # Note, that this is not yet available as a shared library.

@@ -148,7 +146,7 @@
 AC_SUBST(NETLIBS)

 # For src/libassuan-config.in
-LIBASSUAN_CONFIG_LIB="-lassuan"
+LIBASSUAN_CONFIG_LIB="-lassuan1"
 LIBASSUAN_CONFIG_CFLAGS=""
 LIBASSUAN_CONFIG_API_VERSION="1"
 LIBASSUAN_CONFIG_THREAD_MODULES=
--- libassuan-1.0.5/tests/Makefile.am.orig	2010-03-30 15:42:49.061919784 +0200
+++ libassuan-1.0.5/tests/Makefile.am	2010-03-30 15:44:46.594515962 +0200
@@ -35,5 +35,5 @@
 
 noinst_HEADERS = common.h
 noinst_PROGRAMS = $(TESTS)
-LDADD =  ../src/libassuan.la  $(NETLIBS) # $(GPG_ERROR_LIBS)
+LDADD =  ../src/libassuan1.la  $(NETLIBS) # $(GPG_ERROR_LIBS)
 
--- libassuan-1.0.5/src/Makefile.am.orig	2010-03-30 16:24:02.254322139 +0200
+++ libassuan-1.0.5/src/Makefile.am	2010-03-30 16:24:42.715316500 +0200
@@ -23,15 +23,15 @@
 MOSTLYCLEANFILES = assuan-errors.c
 
 if HAVE_PTH
-libassuan_pth = libassuan-pth.la
+libassuan1_pth = libassuan1-pth.la
 else
-libassuan_pth = 
+libassuan1_pth = 
 endif
 
 bin_SCRIPTS = libassuan-config
 m4datadir = $(datadir)/aclocal
 m4data_DATA = libassuan.m4
-lib_LTLIBRARIES = libassuan.la $(libassuan_pth)
+lib_LTLIBRARIES = libassuan1.la $(libassuan1_pth)
 include_HEADERS = assuan.h
 
 
@@ -53,13 +53,13 @@
 	assuan-logging.c \
 	assuan-socket.c
 
-libassuan_la_SOURCES = $(common_sources) assuan-io.c
-libassuan_la_LIBADD = @LTLIBOBJS@
+libassuan1_la_SOURCES = $(common_sources) assuan-io.c
+libassuan1_la_LIBADD = @LTLIBOBJS@
 
 if HAVE_PTH
-libassuan_pth_la_SOURCES = $(common_sources) assuan-io-pth.c
-libassuan_pth_la_CFLAGS = $(AM_CFLAGS) $(PTH_CFLAGS)
-libassuan_pth_la_LIBADD = @LTLIBOBJS@ $(PTH_LIBS)
+libassuan1_pth_la_SOURCES = $(common_sources) assuan-io-pth.c
+libassuan1_pth_la_CFLAGS = $(AM_CFLAGS) $(PTH_CFLAGS)
+libassuan1_pth_la_LIBADD = @LTLIBOBJS@ $(PTH_LIBS)
 endif
 
 assuan-errors.c : assuan.h mkerrors
